os/drivers/lcd/Make.defs

```diff
@@ -115,8 +115,16 @@ ifeq ($(CONFIG_LCD_ST7701SN),y)
   CSRCS += mipi_lcd.c
 endif
 
+ifeq ($(CONFIG_LCD_XRES),480)
+ifeq ($(CONFIG_LCD_YRES),800)
+CSRCS += lcd_logo_52_340.c
+endif
+endif
```

Comment:
Actually I name the file as lcd_logo_800_480 before, but Mr. Nam asked me to use 52_340 instead of lcd resolution.
https://github.ecodesamsung.com/TizenRT/Kernel/pull/666#discussion_r1315027

Reply:
Because the file is a logo, I agree to get logo size instead of lcd size.
But, then, how will you add the same logo with different LCD size?

Is it possible to make proper logo buffer regarding to the size in runtime? or at build time?

Reply:
To resolve this issue, I have added a kconfig based on resolution of the logo. So, same logo can be used in different lcd as selected from menuconfig. https://github.com/Samsung/TizenRT/pull/6521/files#:~:text=CONFIG_LCD_LOGO_52_340)%2Cy)

And lcd buffer is created in runtime according to size of lcd. So, 480_800 lcd will create buffer accordingly. So, different lcd will be supported.
https://github.com/Samsung/TizenRT/pull/6521/files#diff-0dcbbbfee32bf5989cf8e2ad4117f850d4a44a336017e1f0e7f7e4ef484d5e2f:~:text=lcd_data_logo%20%3D%20(uint8_t%20*)kmm_malloc(CONFIG_LCD_XRES%20*%20CONFIG_LCD_YRES%20*%202%20%2B%201)%3B